bump_rpm_packaging

```diff
@@ -13,9 +13,14 @@ if [[ $FULLVERSION == *-RC* ]] ; then
 	if [[ FULLVERSION == *-RC1 ]] ; then
 		echo "TODO: bump release"
```

Comment:
For 1.25 we can also move to `rc1` instead of `RC1` to simplify things. The alternative is to implement `release=increment` in obal.

Reply:
I'd prefer moving to `rc1`, less difference between katello and foreman, and `release=increment` isn't that simple (if you don't want to add a changelog just yet)

Reply:
I agree. Debian also uses lowercase and if Katello RPMs also use lowercase, moving Foreman to lowercase would unify them all.